FrameRate/FrameRate.cpp

```diff
@@ -265,16 +265,27 @@ namespace WPEFramework
 
             string sFramerate = parameters["framerate"].String();
 
-            bool success = true;
-            try
-            {
-                device::VideoDevice &device = device::Host::getInstance().getVideoDevices().at(0);
-                device.setDisplayframerate(sFramerate.c_str());
-            }
-            catch (const device::Exception& err)
+            bool success = false;
+            if(sFramerate == "3840x2160x60"
```

Comment:
We cannot hardcode these values since this is a generic plugin that can be used on different devices with different supported display frame rates. The HAL should error out if the DisplayFrameRate is not getting set. Pls reach out to @apatel859 on details on the error returned and return suitable error in the jsonrpc response for setDisplayFrameRate